NuRadioReco/detector/LOFAR/signalchain/RCU_gain_new_5.txt

```diff
@@ -0,0 +1,51 @@
+0.571199977723
```

Comment:
Is there a README somewhere that explains those odd filenames and where they are from?